util/Migrator/DbMigrator.cs

```diff
@@ -34,15 +34,16 @@ public DbMigrator(string connectionString, ILogger<DbMigrator> logger)
 
             using (var connection = new SqlConnection(_masterConnectionString))
             {
+                var databaseName = new SqlConnectionStringBuilder(_connectionString).InitialCatalog;
```

Comment:
Please add a fallback here just in case:

1. Old client through setup doesn't have an InitialCatalog set, OR
2. Connection string is modified later and initial catalog is removed

Either way, since InitialCatalog isn't required for a valid connection string and older clients likely won't have this value set or may not have this value set, or updating the connection string to a new server they may be copy + pasting from a cloud dashboard, etc. w/o that value, we need to fallback to our defaults before executing SQL below.

```suggestion
                var databaseName = new SqlConnectionStringBuilder(_connectionString).InitialCatalog;
                if (string.IsNullOrWhiteSpace(databaseName))
                {
                    databaseName = "vault";
                }
```

Reply:
Added fallback.